iec61131-st/extra_functions.st

```diff
@@ -214,47 +405,75 @@ END_FUNCTION
 ******************************************************************************)
 FUNCTION TOD_TO_LWORD : LWORD
 VAR_INPUT
-    IN: TOD;
+    IN : TOD;
 END_VAR
     TOD_TO_LWORD := IN;
 END_FUNCTION
 
+FUNCTION LTOD_TO_LWORD : LWORD
```

Comment:
for some of these new functions LTOD_.../LDT_.../ tests are missing

Reply:
As far as I know, LTOD/LDT/etc are just aliases for TOD/DT/... Please correct me if I am wrong. I only implemented the LTIME/LTOD/etc... functions because they were requested in addition to the TIME/TOD implementions, in order to ease porting legacy code.
Since the only difference between functions like LTOD_TO_LWORD and TOD_TO_LWORD is the name, is it really necessary to write tests for all of them?